libmamba/include/mamba/core/fsutil.hpp

```diff
@@ -145,9 +149,10 @@ namespace mamba
             {
                 ec.clear();
                 fs::copy_file(from, to, ec);
-                if (!ec)
+                if (ec)
                 {
-                    fs::remove(from, ec);
+                    std::error_code ec2;
+                    fs::remove(to, ec2);
```

Comment:
Not so familiar with error code, but shouldn't we not checking the result here?

Reply:
we don't really care about the result because at this point the operation has already failed.

Reply:
Adding `[[maybe_unused]]` here would clarify that and avoid warnings.